index.js

```diff
@@ -10,8 +10,13 @@
 var isUTF8 = require('is-utf8');
 var isBuffer = require('is-buffer');
 
+function maybeUTF8(buf) {
```

Comment:
I liked it because it makes it clearer that we aren't totally sure if something is utf8 with this test.